cip-constitution-tech/README.md

```diff
@@ -0,0 +1,261 @@
+---
+CIP: ?
+Title: Constitution form
```

Comment:
@rphair you are very good at naming, please let me know what you think of this title vs "Constitution technical form"

Reply:
My first choice would have been `Constitution Process` but I think too many people would assume this meant either an _approval_ process or a Constitution mandated governance process.

`form` is a good choice but it doesn't include the mechanics like hashing or off-chain storage.  The general engineering term `design` would refer to both form _and_ function so my current best idea would be `Constitution Design` (with the second term also capitalised, since there will only be one of these 😜).

Reply:
My first choice would have been `Constitution Process` but I think too many people would assume this meant either an *approval* process or a Constitution mandated governance process.

`form` is a good choice but it doesn't include the mechanics like hashing or off-chain storage. The general engineering term `design` would refer to both form *and* function so my current best idea would be `Constitution Design` (with the second term also capitalised, since there will only be one of these 😜).

Reply:
`Constitution Design` I feel implies more of the structure of the content of the Constitution

Reply:
Settled on `Constitution specification`, thanks for the suggestion @microproofs